app/src/main/java/org/fossasia/openevent/app/event/detail/ChartMutator.java

```diff
@@ -0,0 +1,73 @@
+package org.fossasia.openevent.app.event.detail;
+
+
+import android.graphics.Color;
+import android.graphics.Paint;
+import android.support.annotation.ColorInt;
+import android.support.annotation.ColorRes;
+
+import com.db.chart.model.LineSet;
+import com.db.chart.renderer.AxisRenderer;
+import com.db.chart.view.LineChartView;
+
+import org.fossasia.openevent.app.R;
+import org.fossasia.openevent.app.data.contract.IUtilModel;
+
+public class ChartMutator {
+    private IUtilModel utilModel;
+
+    private Paint gridPaint = new Paint();
+
+    private LineSet freeSet = new LineSet();
+    private LineSet paidSet = new LineSet();
+    private LineSet donationSet = new LineSet();
+
+    protected ChartMutator(IUtilModel utilModel) {
+        this.utilModel = utilModel;
+
+        gridPaint.setColor(Color.parseColor("#992ecc71"));
+        gridPaint.setStrokeWidth(5);
+
+        freeSet = new LineSet(new String[] {"Jan", "Feb", "Mar"}, new float[] {1.2f, 3.6f, 4.5f});
```

Comment:
This is just mock data, we won't do month wise analysis, we will do aggregate analysis of total sales over the time